mne/viz/_3d.py

```diff
@@ -470,6 +490,126 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     -------
     fig : instance of mlab.Figure
         The mayavi figure.
+
+    See Also
+    --------
+    mne.viz.plot_bem
+    """
+    return plot_alignment(info, trans=trans, subject=subject,
+                          subjects_dir=subjects_dir, source=source,
+                          coord_frame=coord_frame, meg_sensors=meg_sensors,
+                          eeg_sensors=eeg_sensors, dig=dig, ref_meg=ref_meg,
+                          ecog_sensors=ecog_sensors, head=head, brain=brain,
+                          skull=skull, src=src, mri_fiducials=mri_fiducials,
+                          bem=bem, verbose=verbose)
+
+
+@verbose
+def plot_alignment(info, trans='auto', subject=None, subjects_dir=None,
```

Comment:
it seems you can pass 'outer_skin' to source and to skull. That's weird.

I propose the following signature:

def plot_alignment(info, trans='auto', subject=None, subjects_dir=None,
                                surfaces=('head',), coord_frame='head',
                                meg=('helmet', 'sensors'),  # 'helmet' | 'sensors' | None | 'ref' | list
                                eeg='original',  # None | 'original' | 'projected' | list
                                dig=False, 
                                ecog=True,
                                src=None,
                                mri_fiducials=False,
                                bem=None,  # can be BEM of sphere
                                verbose=None)

wdyt?